src/utility/Constants.js

```diff
@@ -21,6 +21,15 @@ export const NOTIFICATION_PERMISSIONS = Object.freeze({
   DENIED: 'denied',
 })
 
+export const WAIVER_LINKS = Object.freeze({
+  COVID:
+    'https://docs.google.com/document/d/1td2BgcwRT2CTYdUrzHji7onxpoVl8UjMiEuqIIUK7CM/edit?usp=sharing%5C',
```

Comment:
We should remove the trailing space in case that causes issues, but that should be fine